src/fragments/start/getting-started/flutter/setup.mdx

```diff
@@ -10,20 +10,20 @@ flutter create amplified_todo
 
 ### Add Amplify to your application
 
-Amplify for Flutter is distributed via **pub.dev**.
+Amplify for Flutter is distributed via [pub.dev](pub.dev).
 
 1. From your project root directory, find and modify pubspec.yaml and add the Amplify plugins to the project dependencies.
 
     ```yaml
     environment:
-      sdk: ">=2.12.0 <3.0.0"
+      sdk: ">=2.17.3 <3.0.0"
 
     dependencies:
       flutter:
         sdk: flutter
 
-      amplify_flutter: ^0.5.0
-      amplify_datastore: ^0.5.0
+      amplify_flutter: ^0.5.1
```

Comment:
same comment about using v 0.6.0 @salihgueler

Reply:
This is also updated already.